src/scripts.rs

```diff
@@ -827,7 +839,7 @@ macro_rules! cpfp_descriptor {
         // Obviously returning rust-miniscript's error here is really ugly, but hopefully
         // this is just a temporary solution (see
         // https://github.com/rust-bitcoin/rust-miniscript/pull/282)
-        if $managers.len() > 20 {
+        if $managers.len() > MAX_MANAGERS {
```

Comment:
This was an ugly workaround because of a bug in rust-miniscript (https://github.com/rust-bitcoin/rust-miniscript/pull/282). Since we now explicitly disallow deployment configs with more than 20 managers this should not return a `PolicyCompilation` compilation error.